Common/Vulkan/VulkanImage.cpp

```diff
@@ -76,11 +76,8 @@ bool VulkanTexture::CreateDirect(VkCommandBuffer cmd, int w, int h, int numMips,
 
 	if (allocator_) {
 		offset_ = allocator_->Allocate(mem_reqs, &mem_, Tag());
-		if (offset_ == VulkanDeviceAllocator::ALLOCATE_FAILED) {
-			vkDestroyImage(vulkan_->GetDevice(), image_, nullptr);
-			ELOG("Image memory allocation failed (mem_reqs.size = %d)", (int)mem_reqs.size);
-			return false;
-		}
+		// Destructor will take care of the image.
+		return false;
```

Comment:
Wait, shouldn't this be inside the ALLOCATE_FAILED if?

-[Unknown]

Reply:
Oops what happened here.. sloppy editing. I'll fix soon.